lib/reporters/cli/index.js

```diff
@@ -288,6 +299,31 @@ PostmanCLIReporter = function (emitter, reporterOptions, options) {
 
 _.assignIn(PostmanCLIReporter, {
 
+    parseSingleRequestStatistics (run) {
+        const summaryTable = new Table({
+                style: { head: [] },
+                colWidths: [40]
+            }),
+            timings = run.timings,
+            transfers = run.transfers;
+
+        // add row to show total data received
+        // eslint-disable-next-line max-len
+        transfers && summaryTable.push([colors.green('Data received: %s (approx)'), util.filesize(transfers.responseTotal)]);
```

Comment:
@raghavbhatia336 Here we are not formatting the string but simply adding a row to the table. Hence there is no need for `%s`. An alternative to the above line could be
```js
summaryTable.push([colors.green('Data received:'), util.filesize(transfers.responseTotal) + ' (approx)']);
```